src/plugins/crypto/crypto.c

```diff
@@ -200,7 +217,15 @@ void elektraCryptoTeardown()
  */
 int elektraCryptoHandleCreate(elektraCryptoHandle **handle, KeySet *config, Key *errorKey)
 {
+#ifdef ELEKTRA_CRYPTO_API_GCRYPT
 	return elektraCryptoGcryHandleCreate(handle, config, errorKey);
+#endif
```

Comment:
there should be a "else", otherwise you could get two return in the function (which results to non-reachable code)